wiki/Manual.md

```diff
@@ -5520,6 +5520,98 @@ Returns
 }
 ```
 
+
+## Proxy
+
+In some specific cases you may want a proxy, when:
+- Exchange is not available in your location
+- Your IP is forbidden by exchange
+- You experience random restriction by exchange, like [DDoS protection by Cloudflare](https://docs.ccxt.com/#/?id=ddos-protection-by-cloudflare-incapsula)
+
+However, beware that each added intermediary might add some latency to requests.
+
+### Supported proxy types
+CCXT supports the following proxy types:
+
+#### proxyUrl
```

Comment:
@ttodua should we add a note like "the one used to setup a CORS proxy" ?

Reply:
added